pennylane/decomposition/resources.py

```diff
@@ -530,3 +560,21 @@ def _controlled_x_rep(  # pylint: disable=too-many-arguments, too-many-positiona
         num_work_wires=num_work_wires,
         work_wire_type=work_wire_type,
     )
+
+
+def auto_wrap(op_type):
```

Comment:
Is this supposed to appear in the docs? Any reason why it is not private anymore?